src/UI/CsobControl.php

```diff
@@ -100,7 +100,13 @@ public function pay()
 		if ($payment->getOriginalPayId()) {
 			$response = NULL;
 			try {
-				$response = $this->client->paymentRecurrent($payment);
+				if ($this->client->isRecurrentPaymentSupported()) {
+					$response = $this->client->paymentRecurrent($payment);
+
+				} else {
+					$initResponse = $this->client->paymentOneclickInit($payment);
```

Comment:
`createPayment` merely creates and sets up the `Payment` object, `paymentInit` is called separately in the [other branch of this condition](https://github.com/Kdyby/CsobPaygateNette/pull/5/files#diff-07614aa4ce077262be1c7f0b078f2111R130). So yeah, I am pretty sure :)